### DISCUSSION TITLE
Spawn bot for LFG

### DISCUSSION THREAD

**Discussion by ilkashpilkerman** [trickerer/Trinity-Bots#902] [Q&A] [answered]
Hi there. 
Will bots spawn automatically when registering specific or random dungeons in LFG, as it happens on BG?
I turned on everything I needed in the settings, but nothing happens when registering the LFG.
Does autocomplete only work on BG?
Regards

**Comment by trickerer** [comment, marked as answer]
No, bots will not spawn automatically to fill up LFG query. Only hired bots are allowed to enter dungeons; in fact bots never actually get registered in LFG queue as players do, they just get automatically counted along with the player if they are in group.
BG bots were only implemented to enable small servers to have functional BGs, dungeon finder can be enabled for everyone by simply increasing the bots limit to 4 per player.

**Reply by ilkashpilkerman** [reply]
Got you. Thanks!
